### PR TITLE
bamutil 1.0.13

### PR DESCRIPTION
**Pull request by pjvandehaar**
Updates to a newer version. Uses sha256.

### CODE REVIEW

File: bamutil.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Bamutil < Formula
   homepage "http://genome.sph.umich.edu/wiki/BamUtil"
```
Comment (dpo) [inline bamutil.rb]: Any chance you could add a `desc` field with a brief description of the software?

**Comment by iml** [comment]
@BrewTestBot test this please.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by pjvandehaar** [comment]
Should I squash these?  Why aren't they merged?

**Comment by iml** [comment]
Thanks! Sorry for the delay.

**Comment by pjvandehaar** [comment]
@iml Thanks!

**Pull request closed without merging**